CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## UNRELEASED
+
+* Add **Other tools** section in README + [Mega-Linter](https://nvuillam.github.io/mega-linter/)
```

Comment:
Does adding a link to a README warrant a changelog entry? I kind of like the idea an unreleased section for the changelog, but it would probably be preferable to generate a changelog at release time instead and avoid as much manual editing as possible.

Reply:
Do you want me to remove changelog editing ?
About the "unreleased" part, it's from https://keepachangelog.com/en/1.0.0/

Reply:
Not my call on whether you should remove it, I'm just providing input as an interested party.

Pretty websites make lot's of best practice claims I don't agree with. I like the idea but not the effort involved with manually updating a changelog with each commit. There's probably a sweet spot where CI maintains a realtime changelog (I'm near certain there are tools for it) from *good* commit messages. I think policing commit message quality is more worthwhile. Worst case with high quality messages is curating a list at release time.

Reply:
> I think policing commit message quality is more worthwhile. Worst case with high quality messages is curating a list at release time.

I totally second that. I think changelog should keep being a software change log. Updating the readme is doc, adding doc on the doc is a bit overkill IMO. Good commit message is the key to me: https://chris.beams.io/posts/git-commit/

Reply:
Removed update on changelog :)